.cargo/config.toml

```diff
@@ -8,11 +8,9 @@
 rustflags = [
   "-Wmissing_docs",
   "-Wmissing_debug_implementations",
-  "-Wmissing_copy_implementations",
   "-Wtrivial_casts",
```

Comment:
Terraform will overwrite this next time it's updated, will break the pipeline. Would recommend restoring this file to default and addressing the warnings in build.rs and through allow statements

Reply:
[686a4cb](https://github.com/Arrow-air/svc-pricing/pull/2/commits/686a4cbfa3101ad1cb790b134423e597bc692766)

Very interesting. All I did was to add these flags back and the github actions are not complaining. NO idea if it's a false positive.